test/FunctionalTests/Microsoft.OData.Core.Tests/ODataMessageWriterTests.cs

```diff
@@ -999,10 +1003,54 @@ public async void DisposeAsync_Should_Dispose_Stream_Asynchronously_AfterWriting
 
             Assert.True(stream.Disposed);
         }
+
+        [Fact]
+        public void WriteMetadataDocument_WorksForLegacyScaleAndSridVariables()
+        {
+            // Arrange
+            IEdmModel edmModel = GetLegacyScaleAndSridModel();
+
+            // Act
+            string payload = this.WriteAndGetPayloadForLegacyScaleAndSridVariables(edmModel, true, omWriter =>

```

Comment:
Not clear what the `true` represents.

Reply:
It's used to determine whether to set the compatibility flag or not. If true, the flag is set,, if false it is not set.